### DISCUSSION TITLE
How to open Desktop in Sandboxie

### DISCUSSION THREAD

**Discussion by fuqifacai** [sandboxie-plus/Sandboxie#4109] [General]
It seems run a exe by default in sandboxie, I just wonder is that possible to run Windows desktop instead to run in sandboxie, 
then I can choose which exe that I want to run ?

**Comment by bastik-1001** [comment]
By default, you have to tell Sandboxie, what is supposed to be launched under its supervision, unless you use force rules or the file is located within a sandbox.

Please try to uninstall it and start with a fresh installation of Sandboxie, so you don't have that behavior, which is not the way it is out of the box.

**Comment by fuqifacai** [comment]
I've tried uninstall and reinstall it , however still not understand what's the goal ? 
My goal is to run one windows desktop in one sandboxie  , is that possible ? 
so far I only can run process in it .

**Reply by bastik-1001** [reply]
I misunderstood what you meant. There is a feature in development, but it has not been released in any form, yet. #4019 is related to it.

**Reply by fuqifacai** [reply]
Seems relative. 
Thanks for your info.

**Comment by love-code-yeyixiao** [comment]
What the “Windows Desktop” mean?



---Original---
From: "Victor ***@***.***&gt;
Date: Thu, Jul 25, 2024 10:50 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [sandboxie-plus/Sandboxie] How to open Desktop in Sandboxie(Discussion #4109)




 
I've tried uninstall and reinstall it , however still not understand what's the goal ?
 My goal is to run one windows desktop in one sandboxie  , is that possible ?
 so far I only can run process in it .

**Reply by fuqifacai** [reply]
Windows Desktop means a window environment with a typical desktop as default UI.
